dissect/target/plugins/os/unix/linux/fortios/_os.py

```diff
@@ -64,27 +105,47 @@ def create(cls, target: Target, sysvol: Filesystem) -> FortigatePlugin:
         if (rootfs_ext_tar := sysvol.path("rootfs-ext.tar.xz")).exists():
             target.fs.add_layer().mount("/", TarFilesystem(rootfs_ext_tar.open("rb")))
 
+        # Filesystem mounts can be discovered in the FortiCare debug report
+        # or using ``fnsysctl ls`` and ``fnsysctl df`` in the cli.
         for fs in target.filesystems:
-            # TODO: Figure out the other partitions
-            # TODO: How to determine /data2?
-            if fs.__type__ == "ext" and fs.extfs.volume_name.startswith("LOGUSEDX"):
+            # log partition
+            if fs.__type__ == "ext" and (
+                fs.extfs.volume_name.startswith("LOGUSEDX") or fs.path("/root/clog").is_symlink()
+            ):
                 target.fs.mount("/var/log", fs)
 
+            # EFI partition
+            if fs.__type__ == "fat" and fs.path("/EFI").exists():
+                target.fs.mount("/boot", fs)
+
+            # data2 partition
+            if fs.__type__ == "ext" and fs.path("/new_alert_msg").exists() and fs.path("/template").exists():
+                target.fs.mount("/data2", fs)
+
         return cls(target)
 
     @export(property=True)
     def hostname(self) -> str | None:
+        """Return configured hostname"""
         try:
-            return self.config["system"]["global"]["hostname"][0]
+            return self._config["global-config"]["system"]["global"]["hostname"][0]
         except KeyError:
             return None
 
     @export(property=True)
     def ips(self) -> list[str]:
+        """Return ip addresses of configured interfaces"""
         result = []
 
         try:
-            for conf in self.config.system.interface.values():
+            # FortiOS 6 and 7 (from global_system_interface.gz)
+            for key, iface in self._config["interfaces"].items():
+                if not key.startswith("port"):
+                    continue
+                result += [ip for ip in iface["ip"] if not ip.startswith("255")]
+
+            # Other versions
+            for conf in self._config["global-config"]["system"]["interface"].values():
```

Comment:
Maybe put these in separate try/excepts? Not sure if `interfaces` always exists.

Reply:
Added in [e765ee2](https://github.com/fox-it/dissect.target/pull/509/commits/e765ee29af0ff2bb8f524c6c5eb01eb7932f9ff6).